packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx

```diff
@@ -136,6 +143,7 @@ const useModelOrganization = () => {
     setCurrentTheme,
     showRelationDetails,
     currentProject,
+    currentResourcesData,
```

Comment:
What is this? Should it be `currentResourcesStorageData` ?

Reply:
Yes, Fixed